grafanalib/core.py

```diff
@@ -11,6 +11,11 @@
 import math
 from numbers import Number
 import warnings
+import re
+
+
+class ParseJsonException(Exception):
+    pass
```

Comment:
Any particular reason for raising exception vs returning None? I don't have an opinion on which is better.

Reply:
There is actually one more option, return the data itself (and this actually makes thing work for a roundtrip).
I went with rising as a signal to the user that something needs to be implemented (that's how I caught `Table`).